blockchain/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dependencies": {
+    "truffle": "^5.5.25"
+  },
+  "devDependencies": {
+    "@nomicfoundation/hardhat-chai-matchers": "^1.0.2",
+    "@nomicfoundation/hardhat-toolbox": "^1.0.2",
+    "@nomiclabs/hardhat-ethers": "^2.1.0",
+    "@openzeppelin/contracts": "^4.7.2",
+    "chai": "^4.3.6",
+    "ethers": "^5.6.9",
+    "hardhat": "^2.10.1"
+  }
+}
```

Comment:
I'm commenting in this file, but maybe you should use yarn, since the other modules are using it. With this done `package-lock.json` will no longer exist only `yarn.lock`

Reply:
to do this, just delete `package-lock.json` and run `yarn`